docs/about.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;1,900&display=swap"
+      rel="stylesheet"
+    />
+
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+    />
+    <link rel="stylesheet" href="./css/main.css" />
+    <title>Ibhicheckesho | About</title>
+  </head>
+  <body>
+    <header id="header-about">
+      <div class="container">
+        <nav id="main-nav">
+          <div class="container">
+            <div id="logo">
+              <a href="index.html"
+                ><img src="./images/logo.png" alt="Watubaki-logo"
+              /></a>
+            </div>
+            <div class="baga-icon-container">
+              <input id="baga-toggler" type="checkbox" />
+              <div class="baga-icon">
+                <div></div>
+              </div>
+            </div>
+            <ul class="nav-menu">
+              <li><a class="nav-menu-link" href="./index.html">Home</a></li>
+              <li><a class="nav-menu-link" href="./about.html">About</a></li>
+            </ul>
+          </div>
+        </nav>
+        <section id="header-content">
+          <h1>
+            <span id="header-content-span">Hello! W'anna Lough? Welcome</span
+            ><span id="header-content-span1">Watubaki</span
+            ><span id="header-content-span2"> &nbsp; Standup Comedy</span
+            ><span id="header-content-span3">March 2022</span>
+          </h1>
```

Comment:
- [ OPTIONAL] Please consider following the design by using the required `background-color` on the h1. This is to make sure that you follow the design guideline and replicate the design to look almost identical to the design guidelines. 

Design guideline:
<img width="406" alt="Loom_Bzg357bceb" src="https://user-images.githubusercontent.com/81468675/147156690-23809f0d-335c-470e-94d2-59b153df040d.png">

Yours:

![cap1](https://user-images.githubusercontent.com/57562869/152233003-937a8b73-d52a-439d-bb0b-facd0315a6d4.png)